service/downloader/client/downloader_client.go

```diff
@@ -86,3 +86,14 @@ func (client *DownloaderClient) ListBucketReadRecord(ctx context.Context, bucket
 	}
 	return resp, nil
 }
+
+// GetSpByAddress get sp by sp address
+func (client *DownloaderClient) GetSpByAddress(ctx context.Context, req *types.GetSpByAddressRequest, opts ...grpc.CallOption) (*types.GetSpByAddressResponse, error) {
```

Comment:
GetSpByAddressRequest change to bucketname?
The specification for client is to be seem to a sdk, it will onvenient caller call

Reply:
fixed